types/npm-profile/index.d.ts

```diff
@@ -40,6 +40,47 @@ export function createToken(
     options?: Options,
 ): Promise<Token>;
 
+/**
+ * Creates a new user on the server along with a fresh bearer token for future authentication as this user.
+ * This is what you see as an authToken in an .npmrc.
+ * @async
+ */
+export function adduser(opener: (url: string) => Promise<void>, prompter: (creds: ProfileAuthCredentials) => Promise<ProfileAuthCredentials>, opts: Options | {}): Promise<ProfileAuthToken>;
```

Comment:
```suggestion
export function adduser(opener: (url: string) => Promise<void>, prompter: (creds: ProfileAuthCredentials) => Promise<ProfileAuthCredentials>, opts?: Options): Promise<ProfileAuthToken>;
```
this is a notation used here for optional parameter (I assume this is the intent here)

Reply:
I think your suggestion is right, yes, this is an optional parameter.